### DISCUSSION TITLE
WREG-DT (News Channel 3) Memphis TN

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#761] [Q&A]
### Channel Name

WREG-DT (News Channel 3)

### Channel ID

WREGDT1.us

### Country

USA, (Memphis, Tennessee)

### Language

English

### Website

https://www.wreg.com/

**Comment by Central302** [comment]
https://tkx.mp.lura.live/rest/v2/mcp/video/adstQ4RVd494gOvJ?anvack=26qYwQVEb2OxXS2W8pu6ZHDRpVdyMgG1&anvtrid=wbd15a4dc4bc0e26cd4a82ac5aa1adb90&rtyp=fp&X-Anvato-Adst-Auth=XiL70WwTu5EgJQTNhoVtXVZfJ5Obwbvb3bvEEd%2FgVBVOypNeSzu2XnBZKEdv8kDCk%2B%2Fnuwau6xEfGzNGb%2BWoTg%3D%3D

In Stort : https://tkx.mp.lura.live/rest/v2/mcp/video/adstQ4RVd494gOvJ?anvack=26qYwQVEb2OxXS2W8pu6ZHDRpVdyMgG1

[Note] Nexstar Owned Station - Anvato.
Need a Magic.

**Comment by BellezaEmporium** [comment]
> https://tkx.mp.lura.live/rest/v2/mcp/video/adstQ4RVd494gOvJ?anvack=26qYwQVEb2OxXS2W8pu6ZHDRpVdyMgG1&anvtrid=wbd15a4dc4bc0e26cd4a82ac5aa1adb90&rtyp=fp&X-Anvato-Adst-Auth=XiL70WwTu5EgJQTNhoVtXVZfJ5Obwbvb3bvEEd%2FgVBVOypNeSzu2XnBZKEdv8kDCk%2B%2Fnuwau6xEfGzNGb%2BWoTg%3D%3D
> 
> In Stort : https://tkx.mp.lura.live/rest/v2/mcp/video/adstQ4RVd494gOvJ?anvack=26qYwQVEb2OxXS2W8pu6ZHDRpVdyMgG1
> 
> [Note] Nexstar Owned Station - Anvato. Need a Magic.

I should really think about making a streamlink plugin for those anvato links when I'll have time.

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
